flopy/mf6/data/mfdataarray.py

```diff
@@ -323,6 +323,71 @@ def data(self):
         """Returns array data.  Calls get_data with default parameters."""
         return self._get_data()
 
+    def to_geodataframe(self, gdf=None, name=None, forgive=False, truncate_attrs=False, **kwargs):
+        """
+        Method to add an input array to a geopandas GeoDataFrame
+
+        Parameters
+        ----------
+        gdf : GeoDataFrame
+            optional GeoDataFrame object
+        name : str
+            optional attribute name, default uses util2d name
+        forgive : bool
+            optional flag to continue if data shape not compatible with GeoDataFrame
+        truncate_attrs : bool
```

Comment:
I think `GeoDataFrame.to_file()` already truncates for you when writing to shapefile, is this parameter necessary? It seems like having our own API for this would be useful if we let the user customize the truncation but otherwise redundant?